SportHub/src/modules/SearchDropdownList/components/SearchDropdownList.js

```diff
@@ -124,15 +127,11 @@ const SearchDropdownList = ({ setIsContentSearch, setContentSearchValue }) => {
           />
         )}
         PaperComponent={({ children }) => (
-          <Paper
-            style={{
-              width: "50vw",
-              maxWidth: "none",
-              marginTop: 3,
-              backgroundColor: "rgba(0, 0, 0, .0)",
-              borderRadius: 0,
-              boxShadow: "none"
-            }}>{children}</Paper>
+          showSuggestions && inputValue && articles.length !== 0 // removing shadows when nothing printed/no results
```

Comment:
can you refactor this into method showShadows() and move the comment to that method